app/scripts/directives/mobileClientDownloadUrl.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+(function() {
```

Comment:
This might be better named `mobileClientDownloadLinks` as it is going to print out a number of links, not just a url (string).  

See [this comment](https://github.com/openshift/origin-web-console/pull/3020/files#r196550463) as well.